templates/foreman_gutterball.yaml.erb

```diff
@@ -0,0 +1,2 @@
+<%# for foreman_gutterball plugin %>
```

Comment:
I would consider namespacing your settings, e.g.:

```
:foreman_gutterball:
  :url:
```

If you don't namespace then `url` gets put into the top level settings namespace and could be over written easily. This way you can access SETTINGS[:foreman_gutterball][:url] and it's clear what 'url' is related to and les slikely to be over written.